arc/aws/391835788720/us-east-1/lf-arc-prod-eks.tf

```diff
@@ -0,0 +1,76 @@
+module "pytorch_arc_prod_eks" {
+  source  = "terraform-aws-modules/eks/aws"
+  version = "~> 20.37"
+
+  cluster_name    = "lf-arc-prod"
+  cluster_version = "1.33"
+
+  cluster_endpoint_public_access = true
+
+  # When bootstraping set to "true" below. Then once the cluster is
+  # created and pytorch-arc-admin role is added, set to "false".
+  enable_cluster_creator_admin_permissions = true
+
+  cluster_compute_config = {
+    enabled    = true
+    node_pools = ["general-purpose"]
+  }
+
+  vpc_id     = module.arc_runners_vpc.vpc_id
+  subnet_ids = module.arc_runners_vpc.private_subnets
+
+  tags = {
+    Environment = var.arc_prod_environment
+  }
+}
+
+################
+# EKS - Access #
+################
+
+resource "aws_eks_access_entry" "admin_role" {
+  cluster_name      = module.pytorch_arc_prod_eks.cluster_name
+  principal_arn     = aws_iam_role.pytorch_arc_admins.arn
```

Comment:
Instead of calling it `pytorch-arc-admins`  maybe something more generic such as `pytorch-ci-admins` because we likely won't have too many different groups of people who need different levels of access. Try to keep it simple.